RNDR_Watchdog_DualUse.ps1

```diff
@@ -399,14 +399,14 @@ Function Stop-Processes {
 
 # Function to download and replace the current version with the latest version from Github repository
 Function Download-Latest-Watchdog {
-

```

Comment:
not sure why these show up as changed... Could be some entity character change?
These changes didn't show up in VSCode and when pushing.

Reply:
Could be this? https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings

Reply:
For cross-platform compatibility I suggest:
`$ git config --global core.autocrlf input` on mac/linux
`$ git config --global core.autocrlf true` on windows